api/cronTasks/cronTasks.js

```diff
@@ -17,6 +17,18 @@ const addTotalPointsToChildren = async () => {
     }
 }
 
+const resetTable = async(table) =>{
+    const IDs = await getIDs(table)
+    await deleteAll(table, IDs)
+}
+
+const deleteAll = async(table, IDs) =>{
+    console.log(IDs)
```

Comment:
I like leaving console logs in code, but make sure that they're commented out for PR's & ideally labeled too. Something like: `console.log("cronTasks ID's from deleteAll: -------->", IDs)` or however you wanna label & format is always appreciated.